docs/apache-airflow/logging-monitoring/logging-tasks.rst

```diff
@@ -113,14 +113,14 @@ See :doc:`../modules_management` for details on how Python and Airflow manage mo
 External Links
 --------------
 
-When using remote logging, users can configure Airflow to show a link to an external UI within the Airflow Web UI. Clicking the link redirects a user to the external UI.
+When using remote logging, you can configure Airflow to show a link to an external UI within the Airflow Web UI. Clicking the link redirects you to the external UI.
```

Comment:
The PR looks good, but why change `users` to `you`?

Reply:
There were 2 different voices in the document. Most of the document was using second-person and I thought it was better to have a single voice across the doc.

Reply:
Cool -- Let's follow https://developers.google.com/style/person